facebook-for-woocommerce.php

```diff
@@ -127,7 +129,7 @@ public function __wakeup() {
 	 */
 	public function init_plugin() {
 
-		if ( ! $this->plugins_compatible() ) {
+		if ( ! Checker::instance()->is_compatible( __FILE__, self::PLUGIN_VERSION ) ) {
```

Comment:
This probably means we can remove plugins_compatible as this is a private method and it's no longer used :-)